frontend/ui/lib/widgets/autocomplete_text_field_widget.dart

```diff
@@ -1,2 +1,62 @@
-export 'package:tripmeout/widgets/autocomplete_text_field_widget_stub.dart'
-    if (dart.library.html) 'package:tripmeout/widgets/autocomplete_text_field_widget_html.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_typeahead/flutter_typeahead.dart';
+import 'package:tripmeout/services/places_services.dart';
+import 'package:tripmeout/model/place.dart';
+import 'package:flutter/src/widgets/basic.dart' as basic;
+
+class MapsApiPlacesTextFieldWidget<T> extends StatefulWidget {
```

Comment:
nit: Nothing in this file is actually parameterized, like everything is parameterized because `_OnClick<T>` is parameterized, but `_OnClick` has no reason to be parameterized.